src/main/java/mcnutty/music/get/ProcessQueue.java

```diff
@@ -11,16 +11,29 @@
     public ArrayList<QueueItem> bucket_played;
     public ArrayList<QueueItem> bucket_youtube;
 
+    private static final int max_buckets = 4;
+
```

Comment:
IMO this should be a command line/configuration file option if we can make it one.